client/app/lib/components/core/Note.jsx

```diff
@@ -9,14 +9,19 @@ const translations = defineMessages({
   },
 });
 
-const Note = ({ message }) => (
+const colorMap = {
```

Comment:
Just saying, in the future, we may consider creating our own Tailwind presets for colours. For example, `bg-primary`, `bg-success`, etc. So, we organise/use colours not by their identity, but what they mean to us.

If tomorrow we decide to rebrand Coursemology to purple, the class names need not change (because their meanings do not change). Essentially decoupling colours from BLOC. Sounds like a note to self LOL

Reply:
Yeah we definitely should centralise all the color setting in the future (similarly we need to avoid bg-green-xx). However, since we currently dont have it yet, I am just creating it for this file first.